src/components/views/settings/tabs/user/SecurityUserSettingsTab.js

```diff
@@ -255,10 +255,9 @@ export default class SecurityUserSettingsTab extends React.Component {
     _renderIgnoredUsers() {
         const {waitingUnignored, ignoredUserIds} = this.state;
 
-        if (!ignoredUserIds || ignoredUserIds.length === 0) return null;
-
-        const userIds = ignoredUserIds
-            .map((u) => <IgnoredUser
+        const userIds = !ignoredUserIds?.length
+            ? 'You have no ignored users.'
```

Comment:
This needs to be a translated string (`_t` and `yarn i18n`)

Reply:
Made the change. I've amended the commit as it's still a very small diff.